inferentia/qa/setup_test_enviroment_and_test.sh

```diff
@@ -32,9 +32,9 @@ export TRITON_CORE_REPO_TAG=${DEFAULT_REPO_TAG}
 export TRITON_BACKEND_REPO_TAG=${DEFAULT_REPO_TAG}
 export TRITON_THIRD_PARTY_REPO_TAG=${DEFAULT_REPO_TAG}
 export IDENTITY_BACKEND_REPO_TAG=${DEFAULT_REPO_TAG}
-export PYTHON_BACKEND_REPO_TAG=${DEFAULT_REPO_TAG}
+export PYTHON_BACKEND_REPO_TAG=jacky-inferentia-neuron
```

Comment:
probably don't want this in here for approval

Reply:
Yes. This is just to ensure the test run links everything correctly. It is removed.